app/Http/Controllers/api/StudentCollaborationController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers\api;
+
+use App\Http\Controllers\Controller;
+//use Illuminate\Http\Request;
+
+class StudentCollaborationController extends Controller
+{
+    public function __invoke()
```

Comment:
Más allá de que son endpoints fake, no deberían corresponder a un uuid? El campo colaboración se desplega en el detalle particular de cada estudiante.

Reply:
De momento no es necesario Alejandro. Cuando estemos hablando de un endpoint real entonces sí que será necesario pasarle el uuid. Por otro lado se puede añadir el uuid como buena práctica a forma de recordatorio de que éste campo será requerido. 

Up to you.